docs/wiki/tuning/older/3-4-tuning-notes.md

```diff
@@ -37,9 +33,9 @@ This removes spikes and sharp edges that otherwise arise during rapid stick inpu
 
 If you enable item_relax, throttle_boost, or use D setpoint weight above zero, you definitely should activate RC smoothing on the relevant axes, as above.
 
-_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
+_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](docs/wiki/guides/current//SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
```

Comment:
```suggestion
_IMPORTANT_: SBus or FPort users [should stop using OpenTX when using 16 channels](docs/wiki/guides/current/SBus-FPort-and-RC-Smoothing-when-using-OpenTX).
```

Reply:
Thanks, I missed those, and they were not detected during build.  Found them all with search and replace, are fixed now.